{"name": "test_addgroup_failure_004", "status": "failed", "statusDetails": {"message": "AssertionError: 400 != 500", "trace": "self = <testplatform.tests.unit.test_Addgroup.TestAddgroup testMethod=test_addgroup_failure_004>\n\n        def test_addgroup_failure_004(self):\n            response = requests.post('http://127.0.0.1:8000/home/api/groups/addgroup/', json={\n      \"name\": \"Python 学习小组\",\n      \"description\": \"这是一个学习 Python 的小组。\",\n    })\n>           self.assertEqual(response.status_code, 500)\nE           AssertionError: 400 != 500\n\ntestplatform\\tests\\unit\\test_Addgroup.py:56: AssertionError"}, "start": 1744076185629, "stop": 1744076185635, "uuid": "40a7d184-42ee-4da6-b5e4-0f9ad5a17e70", "historyId": "6a55cfb01859984e71520d1e03c38297", "testCaseId": "6a55cfb01859984e71520d1e03c38297", "fullName": "testplatform.tests.unit.test_Addgroup.TestAddgroup#test_addgroup_failure_004", "labels": [{"name": "parentSuite", "value": "testplatform.tests.unit"}, {"name": "suite", "value": "test_Addgroup"}, {"name": "subSuite", "value": "TestAddgroup"}, {"name": "host", "value": "HO"}, {"name": "thread", "value": "28800-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testplatform.tests.unit.test_Addgroup"}]}